core/src/smartcontracts/mod.rs

```diff
@@ -26,6 +28,13 @@ pub trait Execute {
     fn execute(self, authority: &AccountId, wsv: &mut WorldStateView) -> Result<(), Error>;
 }
 
+/// This trait should be implemented for all Iroha Instructions.
+#[allow(clippy::len_without_is_empty)]
```

Comment:
```suggestion

```

And implement `is_empty()`. 

OR, better yet, call it something else, like `instruction_size` (semantically, this is a potentially heterogeneous structure, so it makes more sense to call it size).